packages/config/config/devices/0x0159/smart_plug_16a.json

```
@@ -90,25 +91,198 @@
 		},
 		{
 			"#": "30",
-			"$import": "~/templates/master_template.json#state_after_power_failure_prev_off"
+			"label": "Restore on/off status for Smart plug 16A after power failure",
+			"description": "This parameter determines if on/off status is saved and restored for the Smart plug 16A after power failure.",
+			"valueSize": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Restores last on/off status",
+					"value": 0
+				},
+				{
+					"label": "Does not restore last on/off status",
+					"value": 1
+				}
+			]
 		},
 		{
 			"#": "40",
 			"label": "Power Consumption (Watt) Reporting Threshold",
+			"description": "Choose by how much power consumption needs to increase or decrease to be reported. Values correspond to percentages, so if 20 is set (by default), the device will report any power consumption changes of 20% or more compared to the last reading.",
 			"unit": "%",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 100,
-			"defaultValue": 20
+			"defaultValue": 20,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
 		},
 		{
 			"#": "42",
 			"label": "Power Consumption (Watt) Reporting Interval",
+			"description": "Set value refers to the time interval with which power consumption in Watts is reported (30 – 32535 seconds).",
 			"unit": "seconds",
 			"valueSize": 2,
 			"minValue": 0,
 			"maxValue": 32535,
-			"defaultValue": 0
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "50",
+			"label": "Down value",
+			"description": "Lower power threshold used in parameter no. 52.",
```

For example this would be much better:
```suggestion
			"label": "Lower Power Threshold",
			"description": "Used in parameter 52.",
```
and even the description doesn't feel necessary here, since param 52 refers to this one.